README.md

```diff
@@ -24,3 +24,32 @@ The connector is created automatically when the application is started. See the
 for a list of all available properties.
 
 The connector can be configured using a configuration file. See `localdev` for an example configuration file.
+
```

Comment:
Please add a note that the restore is append-only and no offsets are restored. We don't have segment restore in place yet.